### ISSUE TITLE
Help making string changes in dnf5

### ISSUE THREAD

**Issue by rffontenelle** [rpm-software-management/dnf5#1164]
My c++ is not so good, but I'm basically trying to change some strings replacing argument name with placeholder `{}` to deduplicate strings for translation.

Why am I changing this? Because I found some strings that are very similar, differentiating just 'setopt', 'varval', etc. in the beginning. This duplication could lead the translator to translate differently each time, causing inconsistency.

I made the changes and [pushed to my fork](https://github.com/rffontenelle/dnf5/commit/719db646f363d75421e90b0fcee7a28d913e4582). The string extraction worked as expected: less string to translated. However the build fails with the following message and I don't know the best way to solve this.

```
[ 62%] Building CXX object dnf5daemon-client/CMakeFiles/dnf5daemon-client.dir/main.cpp.o
/home/rafael/dnf5/dnf5daemon-client/main.cpp:154:21: error: character constant too long for its type [-Werror]
  154 |                     'setopt',
      |                     ^~~~~~~~
/home/rafael/dnf5/dnf5daemon-client/main.cpp: In lambda function:
/home/rafael/dnf5/dnf5daemon-client/main.cpp:153:21: error: ‘M_’ was not declared in this scope; did you mean ‘M_E’?
  153 |                     M_("{}: Badly formatted argument value: Last key character cannot be '.': {}"),
      |                     ^~
      |                     M_E
/home/rafael/dnf5/dnf5daemon-client/main.cpp:152:41: error: expected primary-expression before ‘(’ token
  152 |                 throw std::runtime_error(
      |                                         ^
/home/rafael/dnf5/dnf5daemon-client/main.cpp:155:26: error: expected ‘)’ before ‘;’ token
  155 |                     value;
      |                          ^
      |                          )
/home/rafael/dnf5/dnf5daemon-client/main.cpp:152:41: note: to match this ‘(’
  152 |                 throw std::runtime_error(
      |                                         ^
cc1plus: all warnings being treated as errors
make[2]: *** [dnf5daemon-client/CMakeFiles/dnf5daemon-client.dir/build.make:342: dnf5daemon-client/CMakeFiles/dnf5daemon-client.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:3379: dnf5daemon-client/CMakeFiles/dnf5daemon-client.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

**Comment by m-blaha** [comment]
Hi, thanks for helping us!
I've added some comments into your commit to help make the code buildable.

**Comment by rffontenelle** [comment]
Thanks! It is now fixed and contributed as #1178. Closing this help request as completed
